DecipherNow/gm-fabric-dashboard
# feat: Disabled Local Storage persistance of dashboards

Temporarily commenting out local storage functionality. 

To test:
1. Start the dashboard
2. Change the __runtime__ meta tag in `public/index.html` to JVM
3. Verify that you see JVM specific dashboards
4. Change the __runtime__ meta tag in `public/index.html to GOLANG
5. Verify that you see GOLANG specific dashboards
6. Open settings and confirm that the "Reset Dashboards" Section does not render


Instructions to restore local storage functionality is at https://gist.github.com/spmcbride1201/d90a7359cc76ab2036903ba506912867

Resolves #335
